src/ETHStakingYieldOracle.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import { Ownable } from 'aave-v3-core/contracts/dependencies/openzeppelin/contracts/Ownable.sol';
+
+contract ETHStakingYieldOracle is Ownable {
```

Comment:
Lets inherit and override IRateSource once the other PR is merged

Reply:
I'd actually inherit proper interfaces even in this PR. It'll make it clear why `_apr` is internal and then we have public `getAPR `, why there is `onReceiveData` etc. At this point it's a little unclear, where everything comes from. It's a little more complex here than just `latestAnswer` in oracle conract.